packages/types/src/base-api.ts

```diff
@@ -16,6 +16,11 @@ export interface IBaseFileSystemSync {
     watchService: IWatchService
     caseSensitive: boolean
 
+    /**
+     * Synchronously copies src to dest. By default, dest is overwritten if it already exists. Returns undefined
```

Comment:
- no need to specify it returns `undefined`. return type is enough.
- add a word about `flags` parameter?